### DISCUSSION TITLE
Possible to "embed" binary into pkg?

### DISCUSSION THREAD

**Discussion by bc3tech** [microsoft/winget-pkgs#56198] [Q&A] [answered]
I have a tool that I've [submitted to Chocolatey](https://community.chocolatey.org/packages/filesorter) but I'd also like to put it on Winget.

It's a simple exe that's built with a .Net project which I host [here on GitHub](https://github.com/bc3tech/FileSorter). I was hoping to create a winget manifest that had the exe built from `dotnet build` right inside it but it doesn't look like this is possible.

Is there a recommended way to accomplish this?

Thanks

**Comment by Trenly** [comment, marked as answer]
Currently packages must have an installer which installs the binary to the system and adds the registry entries to register the package in the ARP table. However, once microsoft/winget-cli#182 is implemented, then portable binaries will be able to be added

**Reply by bc3tech** [reply]
Thanks for that tip! just subscribed to that issue.
